db/db_basic_test.cc

```diff
@@ -3224,30 +3242,136 @@ TEST_F(DBBasicTest, PointLookupDeadline) {
     // and cause the Get() to fail.
     while (timedout) {
       ReadOptions ro;
-      ro.deadline = std::chrono::microseconds{env->NowMicros() + 10000};
-      fs->SetDelaySequence(
-          ro.deadline, {std::tuple<int, int, IOStatus>{
-                           io_deadline_trigger, 20000, IOStatus::TimedOut()}});
+      if (set_deadline) {
+        ro.deadline = std::chrono::microseconds{env->NowMicros() + 10000};
+      }
+      if (set_timeout) {
+        ro.io_timeout = std::chrono::microseconds{5000};
+      }
+      fs->SetDelayTrigger(ro.deadline, ro.io_timeout, io_deadline_trigger);
 
       block_cache->SetCapacity(0);
       block_cache->SetCapacity(1048576);
 
       std::string value;
       Status s = dbfull()->Get(ro, "k50", &value);
       if (fs->TimedOut()) {
-        ASSERT_EQ(s, Status::TimedOut());
+        if (s != Status::TimedOut()) {
+          ASSERT_EQ(s, Status::TimedOut());
+        }
```

Comment:
`ASSERT_EQ(s, Status::Timedout())` is asserting `s == Statut::Timedout()`. Am I missing anything?

Reply:
Sorry, I had the ```if``` check there for debugging purposes. Will remove it.